__tests__/unittest/build.test.js

```diff
@@ -152,4 +153,41 @@ describe("Build", () => {
         expect(console.log).toHaveBeenCalledTimes(1);
         expect(promise.length).toEqual(0);
     });
+
+    it("should update resource URLs with relative paths and without colunms", () => {
+        const buildClass = new OrdBuildPlugin();
+        const ordDocument = {
+            apiResources: [
+                {
+                    ordId: "sap.sm:apiResource:SupplierService:v1",
+                    resourceDefinitions: [
+                        { url: "/ord/v1/resource1:v1" },
+                        { url: "/ord/v1/resource2:v1" },
+                    ],
+                },
+            ],
+            eventResources: [
+                {
+                    ordId: "sap.sm:eventResource:SupplierService:v1",
+                    resourceDefinitions: [
+                        { url: "/ord/v1/event1:v1" },
+                        { url: "/ord/v1/event2:v1" },
+                    ],
+                },
+            ],
+        };
+        const updatedOrdDocument = buildClass.postProcess(ordDocument);
+        expect(updatedOrdDocument.apiResources[0].resourceDefinitions[0].url).toBe(
+            path.join(
+                BUILD_DEFAULT_PATH,
+                "resource1_v1"
+            )
+        );
+        expect(updatedOrdDocument.eventResources[0].resourceDefinitions[0].url).toBe(
+            path.join(
```

Comment:
Can you please hard code the result to avoid mistakes in the result calculation of the test?